specification/storage/data-plane/Microsoft.StorageDataLake/stable/2018-11-09/DataLakeStorage.json

```diff
@@ -6,172 +6,35 @@
-      "headers": {
-        "x-ms-request-id": {
-          "description": "A server-generated UUID recorded in the analytics logs for troubleshooting and correlation.",
-          "pattern": "^[{(]?[0-9a-f]{8}[-]?([0-9a-f]{4}[-]?){3}[0-9a-f]{12}[)}]?$",
-          "type": "string"
-        },
-        "x-ms-version": {
-          "description": "The version of the REST protocol used to process the request.",
-          "type": "string"
-        }
-      },
-      "schema": {
-        "$ref": "#/definitions/DataLakeStorageError"
-      }
-    }
-  },
-  "paths": {
+  "paths": {},
+  "x-ms-paths": {
     "/": {
       "get": {
-        "operationId": "Filesystem_List",
+        "operationId": "Service_ListFilesystems",
```

Comment:
Are we planning to case this as `Filesystems` instead of `FileSystems`?

Reply:
Typo, supposed to be FileSystems.  I'll fix before I merge.